src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { pipe, toUpper, trim } from 'ramda';
+import PropTypes from 'prop-types';
+
+import { Button } from '../Button';
+import { ButtonColors, ButtonVariants } from '../Button/Button.enums';
+import { Sizes, UserAvatarProps } from './UserAvatar.types';
+import {
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  getRadii,
+  pxToRem,
+} from '../../utils/helpers';
+import { UserAvatarSizes } from './UserAvatar.enums';
+
+const UserAvatarSmall = css`
+  padding: ${pxToRem(4, 2.5)};
+`;
+const UserAvatarMedium = css`
+  padding: ${pxToRem(8, 6.5)};
+`;
+
+const userAvatarSizes = {
+  [UserAvatarSizes.sm]: UserAvatarSmall,
+  [UserAvatarSizes.md]: UserAvatarMedium,
+};
+
+const StyledButton = styled(Button)<{ size: Sizes }>`
+  border-radius: ${getRadii('round')};
+  font-size: ${getFontSize('md')};
+  font-weight: ${getFontWeight('regular')};
+  line-height: ${getLineHeight('md')};
+  ${({ size }) => userAvatarSizes[size]};
+`;
+const normalizeString = pipe(trim, toUpper);
+
+const UserAvatar: React.FC<UserAvatarProps> = ({
+  children,
+  size = UserAvatarSizes.md,
+  ...props
+}) => (
+  <StyledButton
+    color={ButtonColors.primary}
```

Comment:
We should make this with custom (branding) colors, we will change this later in Q3 when wi will change button colors. We will use this in the header and if we make this blue it can look strange in the context of our logo.
For now let's use arbitrary colors and leave TODO comment here to migrate when we change button colors.
```
default = #6641F3
hover = #5737CF
active = #482EAD

Reply:
Just to be sure that you have read the Figma message :D 

@ajkl2533 I went with the primary due to this comment on FIgma: 
https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/DS-Atoms?node-id=17126%3A21530

Known issues:
"the brand color is not supported on interactive elements yet, so we are using the primary colors as a fallback"

So, would you like to go with the new brand ones instead?

Reply:
@dclerici-ssc I'm aware of that comment. But even with that default state is in Brand color. I think we should go with the purple color it seems like the right solution, even if we need to refactor it later.